providers.tf

```diff
@@ -0,0 +1,28 @@
+terraform {
+
+  required_version = "= 1.5.7"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
```

Comment:
sync with @NikolayMarusenko to properly set values

Reply:
We discussed it during the sync, and there were no comments from @NikolayMarusenko.